pages/linux/notify-send.md

```diff
@@ -0,0 +1,15 @@
+# Notify-send
+
+Uses the current desktop environment's notification system to create a notification. This should work on many modern desktop environments such as XFCE, GNOME, and Cinnamon.
```

Comment:
IMO, sentence "This should work on many modern desktop environments such as XFCE, GNOME, and Cinnamon." should not be here. If users are looking for it, then they have this command.